lib/aws_agcod/create_gift_card.rb

```diff
@@ -28,6 +28,10 @@ def claim_code
       @response.payload["gcClaimCode"]
     end
 
+    def expiration_date
+      Time.parse @response.payload["gcExpirationDate"]
```

Comment:
I would suggest that we put this method lazy loaded, like:
`@expiration_date ||= Time.parse @response.payload["gcExpirationDate"]`
So we don't need to call `Time.parse` several times if `expiration_date` is used in multiple places.